yarn.lock

```diff
@@ -2303,6 +2303,11 @@ caseless@~0.12.0:
   version "0.12.0"
   resolved "https://registry.yarnpkg.com/caseless/-/caseless-0.12.0.tgz#1b681c21ff84033c826543090689420d187151dc"
 
+chain-function@^1.0.0:
```

Comment:
What were all of these added for? There are a lot of packages added.

Reply:
I think I had added some packages and then not used them. they should be removed now